cogs/helpers.py

```diff
@@ -89,66 +89,68 @@ async def exam(self, ctx):
 
     @commands.command()
     async def weather(self, ctx):
-        """Retrieves current weather conditions.
-        Data taken from http://weather.gc.ca/city/pages/qc-147_metric_e.html"""
+        """
+        Retrieves current weather conditions.
+        Data taken from http://weather.gc.ca/city/pages/qc-147_metric_e.html
+        """
         def retrieve_string(label):
```

Comment:
shouldn't this be defined outside the weather command? maybe either as a private static method, or as a function in utils?

Reply:
less sure about this one cause its tiny but yeah maybe